README.md

```diff
@@ -16,9 +16,10 @@ This Hugo theme is a dead simple integration between [plausible.io](https://www.
 
 ```toml
 [params.plausible]
-   enable      = true
-   subdomain   = false
-   analytics   = "put-here-your-plausible-analytics-code"
+   enable      = true  # Whether to enable plausible tracking
+   domain   = "example.com"
```

Comment:
Thanks. I get the idea. Nice proposal.

But let me think about the 3 domain names because at the moement it is a bit confusing. But I agree that the names should be self explanating.

And If we can avoid a breaking change it will be better.

Reply:
Yeah i'm not entirely happy with the naming of some of the keys, but it was the best I could come up with.

I think a breaking change is needed, as the previous variable names aren't very descriptive. Some [warnings](https://gohugo.io/functions/errorf) could be put in to migrate to new names, and then make it breaking, but that feels unnecessary at this point, as users should be versioning the theme anyway using submodules or alike.

Reply:
Fair enough. This module is really young publicly. So better break now than later.

I (our you in PR ?) will make it a v1.1.0 and may be put a small warning.